crates/notion-core/src/session.rs

```diff
@@ -174,16 +174,30 @@ impl Session {
     /// Fetch and unpack a version of Node matching the input requirements.
     pub fn install_node(&mut self, version_spec: &VersionSpec) -> Fallible<()> {
         let node_distro = self.fetch_node(version_spec)?.into_version();
+        let success_message = format!(
+            "Installation successful. Default node version set to {}",
```

Comment:
Can we format this to match the rest of the installation/operation output? And the same with the others below?

Reply:
I'm not sure I follow, what installation / operation output do you mean?

Reply:
Sorry; I was meaning that we could match the styling for things like `Fetching` or `error:` and print it with something like:

```
*Success*: installed and set node@{} as default.
```

where `*Success*` should be green instead of wrapped in `*` characters.

Reply:
Ahh, good call, I'll update to do that :+1: